rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/SQSRestServerBuilder.scala

```diff
@@ -57,7 +58,8 @@ case class TheSQSRestServerBuilder(
     generateServerAddress: Boolean,
     sqsLimits: Limits,
     _awsRegion: String,
-    _awsAccountId: String
+    _awsAccountId: String,
+    queueMetadataListener: Option[ActorRef]
```

Comment:
I think there should be a method `withQueueMetadataListener` to add the listener